python/ray/serve/config.py

```diff
@@ -64,7 +64,7 @@ class BackendConfig(BaseModel):
     user_config: Any = None
 
     experimental_graceful_shutdown_wait_loop_s: PositiveFloat = 2.0
-    experimental_graceful_shutdown_timeout_s: PositiveFloat = 20.0
+    experimental_graceful_shutdown_timeout_s: confloat(ge=0) = 20.0
```

Comment:
whoa what is confloat?

Reply:
constrained float. I need it so that we can have >=0